README.md

```diff
@@ -8,6 +8,15 @@ This demo showcases how you can perform the following actions :
 
 - Request Data
 
+
+1. Schema for this send and request data
```

Comment:
The formatting of the list is weird. Try removing the ''1. ".
And reword the phrase like "The following schemas are used by the 'Send Data' and 'Request Data' actions:"

Result should be:
``` md
The following schemas are used by the 'Send Data' and 'Request Data' actions:
- [Coupon](https...)
- [Contact](https...)
- [COVID19 (PCR) Test](https...)
```